xml/System.Diagnostics.Metrics/Measurement`1.xml

```diff
@@ -160,8 +160,8 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="value">The value of the measurement.</param>
-        <param name="tags">A <see cref="T:System.Diagnostics.TagList" /> containing the <see cref="T:System.Collections.Generic.KeyValuePair`2" /> tags associated with the measurement.</param>
+        <param name="value" />
```

Comment:
Bad doc changes, presumably from https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource. This package isn't in the list of packages that doesn't use compiler generated docs though...